### ISSUE TITLE
"dot-ignore" template file absent from CCI project template files

### ISSUE THREAD

**Issue by RupertBarrow** [SFDO-Tooling/CumulusCI#2978]
### Describe the bug

When running `cci project init` we finish with a `Error: dor-ignore`message which indicates the absence of a project template .ignore file

Indeed, after a CumulusCI clean install, the folder '.../pipx/venvs/cumulusci/lib/python3.10/site-packages/cumulusci/files/templates/project' contains not `dot-ignore` file 

### Reproduction steps

1. Clean install Cumulus CI 3.48.1 following the CumulusCI Documentation : 
`
brew install pipx
pipc ensurepath
pipx install cumulusci
`

2. In an SFDX project folder, initialize it for CumulusCI : 
`
cci project init
`
3. Read the 'Error: dot-ignore' error message
4. Get more details : 
`cci error info`
Read the details about the absence of dot-gitignore file
5. run `which cci`
6. In that folder, look for the '.../pipx/venvs/cumulusci/lib/python3.10/site-packages/cumulusci/files/templates/project' subfolder
7. Verify that there is not dot-gitignore or .gitignore file there

### Your CumulusCI and Python versions

CumulusCI version: 3.48.1 (/Users/rupert/.local/bin/cci)
Python version: 3.10.0 (/Users/rupert/.local/pipx/venvs/cumulusci/bin/python)

You have the latest version of CumulusCI.

### Operating System

Mac OS 11.6

### CumulusCI installation method

pipx

**Comment by davisagli** [comment]
Thanks for the report. I confirmed I can reproduce this with cumulusci 3.48.1. The actual error is `jinja2.exceptions.TemplateNotFound: dot-gitignore` and this is the full stack trace:

```
Traceback (most recent call last):
  File "/Users/dglick/.pyenv/versions/cci/bin/cci", line 8, in <module>
    sys.exit(main())
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/cumulusci/cli/cci.py", line 87, in main
    handle_exception(
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/cumulusci/cli/cci.py", line 122, in handle_exception
    raise error
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/cumulusci/cli/cci.py", line 78, in main
    cli(args[1:], standalone_mode=False, obj=runtime)
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/cumulusci/cli/runtime.py", line 183, in new_func
    func(runtime, *args, **kw)
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/cumulusci/cli/project.py", line 213, in project_init
    init_from_context(context, echo=True)
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/cumulusci/cli/project.py", line 228, in init_from_context
    template = env.get_template(name)
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/dglick/.pyenv/versions/3.8.0/envs/cci/lib/python3.8/site-packages/jinja2/loaders.py", line 249, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: dot-gitignore
```

Looks like this is a regression from #2960 because the new file name was not included in the released cumulusci package.

As a workaround until we are able to release a fix for this, you can install cumulusci 3.47.0 (`pipx install cumulusci==3.47.0`)

**Comment by davisagli** [comment]
We've released cumulusci 3.48.2 with a fix for this issue.
